app/lib/features/search/pages/quick_jump.dart

```diff
@@ -18,19 +18,24 @@ class QuickjumpDialog extends ConsumerWidget {
             Routes? route,
             bool push = false,
             String? target,
+            Future<void> Function(BuildContext)? custom,
           }) async {
-            context.pop();
-            if (push) {
-              if (route == null) {
-                await context.push(target!);
-              } else {
-                await context.pushNamed(route.name);
-              }
+            if (custom != null) {
+              await custom(context);
```

Comment:
What is `await custom(context);` ?

Reply:
It's a custom callback to run rather than specific routing - it's the callback we do for the bug reporter to close the quick jumper and take the screenshot before opening the bug reporter